docs/release-notes.rst

```diff
@@ -0,0 +1,3 @@
+.. _release notes:
+
+.. release-notes:: Release Notes
```

Comment:
Thanks for setting this up. As you know, reno can be a pain and I wasn't sure if everything would behave with no notes